static/index.html

```diff
@@ -66,7 +66,7 @@ <h1>Welcome to the Microsoft Campus Community</h1>
 
             <section id="bridgeSection">
                 <div class="sectionDiv">
-                    <p>It is not all about coding. We want you to connect with peers and industry experts. Foster your own network, learn new technologies and accelerate your personal growth. To be a future leader in tech you should be open to share expertise as well as experience and inspire others to live up to their potential.</p>
+                    <p>It is not all about coding. We want you to connect with peers and industry experts. Foster, your network, learn new technologies and accelerate your personal growth. To be a future leader in tech, you should be open to share expertise as well as experience and inspire others to live up to their potential.</p>
```

Comment:
Is it Foster your network, learn new technologies or Foster, your network, learn new technologies?

Reply:
I think the comma is there by mistake. The original one sounds better, here.